trillian/ctfe/configpb/config.proto

```diff
@@ -88,6 +93,13 @@ message LogConfig {
   // log's key and can't sign STHs. Consequently, the log operator must ensure
   // to channel source log's STHs into CTFE.
   bool is_mirror = 12;
+  // custom_handler_prefix if set to a non empty value overrides the global
+  // handler prefix for an individual log. For example this field is set to
+  // "/otherlogs" then a log with prefix "vogon" will make it's get-sth handler
+  // available at "/otherlogs/vogon/ct/v1/get-sth" regardless of what the
+  // global prefix is. Can be set to '/' to make the get-sth handler register
+  // at "/vogon/ct/v1/get-sth".
+  string custom_handler_prefix = 13;
```

Comment:
Can we make this simpler by dropping `custom_handler_prefix` and allowing `/` in the per-log prefix?  That way you could set the global prefix to `"/"` and have per-log prefixes of `"logs/vogon"`, `"otherlogs/dent"`, `"toplevel"` ?

Reply:
Possibly. The advantage of this is that it doesn't change the config validation rules. Plus prefix is used in other places like metrics? so there could be unexpected additional impact of changing that rule.

Reply:
Or is tree id the only thing metrics uses? Not entirely sure.

Reply:
Ah, fair point about `prefix`.  Maybe:
 - rename `custom_handler_prefix` to `override_handler_prefix`
 - move the field up next to `prefix` (if that's allowed for .proto files)

Reply:
Just waiting to see if the updated protoc-gen-go fixes the build error then I'll have another look at it.

Reply:
Done.